AppController/djinn.rb

```diff
@@ -2037,7 +2042,7 @@ def start_new_roles_on_nodes_in_cloud(ips_to_roles)
 
     begin
       new_nodes_info = imc.spawn_vms(num_of_vms, @options, roles, disks)
-    rescue FailedNodeException, AppScaleException => exception
+    rescue Exception => exception
```

Comment:
I think we want to avoid doing these generic catch-all rescues. In this case, though, it might be more effort than it's worth.